WORKSPACE

```diff
@@ -0,0 +1,30 @@
+http_file(
+    name = "xctestrunner",
```

Comment:
Does something external depend on this?

Reply:
Yea it's used from here https://github.com/bazelbuild/rules_apple/blob/a008ce3638d05de4125d462b735da59980d24b68/apple/testing/default_runner/ios_test_runner.bzl#L65-L68 and mentioned in the setup here https://github.com/bazelbuild/rules_apple/blob/a008ce3638d05de4125d462b735da59980d24b68/README.md#quick-setup

Without this you can't run iOS tests

Reply:
Do you know why `rules_apple` wouldn't fetch this dependency? Why it pushes this dependency up on consumers?

Reply:
I submitted https://github.com/bazelbuild/rules_apple/pull/331 for discussion